pkg/url/url.go

```diff
@@ -364,6 +440,86 @@ func List(client *occlient.Client, localConfig *config.LocalConfigInfo, componen
 	return urlList, nil
 }
 
+// ListDockerURL returns all Docker URLs for given component.
+func ListDockerURL(componentName string, envSpecificInfo *envinfo.EnvSpecificInfo) (URLList, error) {
+	client, err := lclient.New()
+	if err != nil {
+		return URLList{}, err
+	}
+	containers, err := dockerutils.GetComponentContainers(*client, componentName)
+	if err != nil {
+		return URLList{}, errors.Wrap(err, "unable to list component container")
+	}
+
+	localURLs := envSpecificInfo.GetURL()
+
+	var urls []URL
+
+	for _, c := range containers {
+		var found = false
+		containerJSON, err := client.Client.ContainerInspect(client.Context, c.ID)
```

Comment:
same as https://github.com/openshift/odo/pull/3086/files#r420746604. It seems we calling it again at https://github.com/openshift/odo/pull/3086/files#diff-1e4fc401677b064040ea6bf41227b6acR497

Reply:
Updated